.golangci.yml

```diff
@@ -0,0 +1,12 @@
+# See https://golangci-lint.run/usage/configuration/
```

Comment:
This one trigger a new CI for PR? I only see github action been triggered?

Reply:
It is run by the Lint job in the GitHub Action workflow